govcd/api_vcd.go

```diff
@@ -29,6 +29,8 @@ func init() {
 	requestCounter = &counter
 }
 
+var minVcdApiVersion = "37.0" // supported by 10.4+
```

Comment:
Extracting default minimum supported API version definition into a global variable so that it is shared between client initialization `NewVCDClient`, but can also be used in unit tests